llvm/unittests/Object/ELFObjectFileTest.cpp

```diff
@@ -984,10 +984,23 @@ TEST(ELFObjectFileTest, ReadPGOAnalysisMap) {
               BrProb:      0xffffffff
           - BBFreq:         1000
             Successors:    []
-)");
+  - Name: .llvm_bb_addr_map_5
+    Type: SHT_LLVM_BB_ADDR_MAP
+  # Link: 0 (by default, can be overriden)
+    Entries:
+      - Version: 2
+        Address: 0x55555
+        Feature: 0x0
+        BBEntries:
+          - ID:            2
+            AddressOffset: 0x0
+            Size:          0x2
+            Metadata:      0x4
+    PGOAnalyses: [{}]
```

Comment:
Is there a test to catch inconsistency between Feature and PGOAnalyses? For example, one where Feature is 0x0 but PGOAnalyses is non-empty? We can add this in a separate PR.

Reply:
We have some where fields are missing when they should be but not one for when they are present but shouldn't. I'll make a PR for that.